proxy/proxyUtils.js

```diff
@@ -55,14 +70,19 @@ const handleSessionExpiry = (proxyRes, proxyResData, req, res, data) => {
       ts: dateFormat(new Date(), 'yyyy-mm-dd HH:MM:ss:lo'),
       params:
       {
-        'msgid': null,
+        'msgid': req.headers['x-request-id'] || '',
         'status': 'failed',
         'err': 'SESSION_EXPIRED',
         'errmsg': 'Session Expired'
       },
       responseCode: 'SESSION_EXPIRED',
       result: {}
     };
+  } else if(error || errorStatus.includes(proxyRes.statusCode)) {
+    edata['message'] = `${req.originalUrl} failed`;
+    logger.info({message: `${req.originalUrl} failed`});
```

Comment:
This is not standard log format. Make sure all logs always follow the standards.